test/powershell/Modules/Microsoft.PowerShell.Utility/Export-Csv.Tests.ps1

```diff
@@ -3,71 +3,69 @@ Describe "Export-Csv" -Tags "CI" {
     $testCsv = Join-Path -Path $TestDrive -ChildPath "output.csv"
 
     AfterEach {
-	Remove-Item $testCsv -Force -ErrorAction SilentlyContinue
+        Remove-Item $testCsv -Force -ErrorAction SilentlyContinue
     }
 
     It "Should be able to be called without error" {
-	{ $testObject | Export-Csv $testCsv } | Should Not Throw
+        { $testObject | Export-Csv $testCsv } | Should Not Throw
     }
 
     It "Should throw if an output file isn't specified" {
-	{ $testObject | Export-Csv -ErrorAction SilentlyContinue } | Should Throw
+        { $testObject | Export-Csv -ErrorAction SilentlyContinue } | ShouldBeErrorId "CannotSpecifyPathAndLiteralPath,Microsoft.PowerShell.Commands.ExportCsvCommand"
     }
 
     It "Should be a string when exporting via pipe" {
-	$testObject | Export-Csv $testCsv
+        $testObject | Export-Csv $testCsv
 
-	$piped = Get-Content $testCsv
+        $piped = Get-Content $testCsv
 
-	$piped[0] | Should Match ".String"
+        $piped[0] | Should be "#TYPE System.String"
     }
 
     It "Should be an object when exporting via the inputObject switch" {
-	Export-Csv -InputObject $testObject -Path $testCsv
+        Export-Csv -InputObject $testObject -Path $testCsv
 
-	$switch = Get-Content $testCsv
+        $switch = Get-Content $testCsv
 
-	$switch[0] | Should Match ".Object"
+        $switch[0] | Should Match ".Object"
     }
 
     It "Should output a csv file containing a string of all the lengths of each element when piped input is used" {
-	$testObject | Export-Csv -Path $testCsv
+        $testObject | Export-Csv -Path $testCsv
 
-	$first    = "`"" + $testObject[0].Length.ToString() + "`""
-	$second   = "`"" + $testObject[1].Length.ToString() + "`""
-	$third    = "`"" + $testObject[2].Length.ToString() + "`""
-	$expected = @("#TYPE System.String", "`"Length`"", $first , $second, $third)
+        $first    = "`"" + $testObject[0].Length.ToString() + "`""
+        $second   = "`"" + $testObject[1].Length.ToString() + "`""
+        $third    = "`"" + $testObject[2].Length.ToString() + "`""
+        $expected = @("#TYPE System.String", "`"Length`"", $first , $second, $third)
 
-	for ( $i = 0; $i -lt $testCsv.Length; $i++)
-	{
-	    $(Get-Content $testCsv)[$i] | Should Be $expected[$i]
-	}
+        for ( $i = 0; $i -lt $testCsv.Length; $i++) {
+         $(Get-Content $testCsv)[$i] | Should Be $expected[$i]
+        }
     }
 
     It "Should be able to use the epcsv alias without error" {
-	{ $testObject | Export-Csv -Path $testCsv } | Should Not Throw
+        { $testObject | epcsv -Path $testCsv } | Should Not Throw
     }
 
     It "Should have the same information when using the alias vs the cmdlet" {
```

Comment:
Please remove the test at all - we have another file for alias tests.

Reply:
Done